META6.json

```diff
@@ -13,7 +13,7 @@
         "runtime": {
             "requires": [
                 "NativeCall",
-                "NativeLibs:ver<0.0.7>:auth<github:salortiz>",
+                "NativeLibs:ver<0.0.7+>:auth<github:salortiz>",
```

Comment:
Shouldn't this [line](https://github.com/CurtTilmes/raku-libcurl/blob/master/lib/LibCurl/EasyHandle.rakumod#L1) also be changed (and any other occurrences).

Another point but I'm not sure about, `DBIish` older versions like `0.5.19` provides `NativeLibs` in the `provides` section, wouldn't this cause conflict when loading   `NativeLibs:ver<0.0.7+>`? that rakudo might load the version provided by `DBIish` since it will have a higher version. I don't know how Rakudo loads the modules but assuming that rakudo decides based on version only, should the version be pined to `0.0.8` instead of `0.0.7+`?

This is not the best solution becuase it will require version update for all modules depend on `NativeLib` when there is anew version of `NativeLib`. may be a better solution is to remove the old DBIish versions that provide `NativeLibs` from the ecosystem then modules depends on `NativeLibs` can just specify the name without `:ver<>` part?